docs/stage/download-raw-data.md

```diff
@@ -33,3 +36,16 @@ Run the SQL:
     DELETE FROM download_raw_errors;
     UPDATE publisher_feed SET raw_next_url=NULL;
     
+
+## To only download some raw data
```

Comment:
The word 'some' here doesn't necessarily mean 'a subset' if you're not looking at this through the lens that we are writing this. 

Maybe "To download a limited set of data"? And then some explanatory text as to why you might want to.